test/13_arraycommon.jl

```diff
@@ -1420,8 +1420,8 @@ end
         @test sprint((io,a)->show(io, "text/plain", a), x) ==
             """
             3-element $CategoricalArray{$(Union{Missing,Int}),1,UInt32}:
-             2      
-             1      
+             2
```

Comment:
I think this makes tests fail.

Reply:
interesting - why on old versions spaces were added?

Reply:
Not sure. Maybe some padding was added on the right?